standalone/default.cfg

```diff
@@ -0,0 +1,393 @@
+
+[DAEMON]
+# set to 1 to run the daemon service within this process;
+# set to 0 if the daemon is run separately (using script joinmarketd.py)
+no_daemon = 1
+
+# Port on which daemon serves; note that communication still
+# occurs over this port even if no_daemon = 1
+daemon_port = 27183
+
+# Currently, running the daemon on a remote host is
+# *NOT* supported, so don't change this variable
+daemon_host = localhost
+
+# by default the client-daemon connection is plaintext, set to 'true' to use TLS;
+# for this, you need to have a valid (self-signed) certificate installed
+use_ssl = false
+
+[BLOCKCHAIN]
+# options: bitcoin-rpc, regtest, bitcoin-rpc-no-history, no-blockchain
+# When using bitcoin-rpc-no-history remember to increase the gap limit to scan for more addresses, try -g 5000
+# Use 'no-blockchain' to run the ob-watcher.py script in scripts/obwatch without current access
+# to Bitcoin Core; note that use of this option for any other purpose is currently unsupported.
+blockchain_source = bitcoin-rpc
+
+# options: signet, testnet, mainnet
+# Note: for regtest, use network = testnet
+network = mainnet
+
+rpc_host = localhost
+# default ports are 8332 for mainnet, 18443 for regtest, 18332 for testnet, 38332 for signet
+rpc_port =
+
+# Use either rpc_user / rpc_password pair or rpc_cookie_file.
+rpc_user = bitcoin
+rpc_password = password
+#rpc_cookie_file =
+
+# rpc_wallet_file is Bitcoin Core wallet which is used for address and
+# transaction monitoring (it is watchonly, no private keys are stored there).
+# It must be created manually if does not exist, see docs/USAGE.md for more
+# information.
+rpc_wallet_file =
+
+## SERVER 1/4) Darkscience IRC (Tor, IP)
+################################################################################
+[MESSAGING:server1]
+channel = joinmarket-pit
+port = 6697
+usessl = true
+
+# For traditional IP (default):
+#host = irc.darkscience.net
+#socks5 = false
+
+# For Tor (recommended as clearnet alternative):
+host = darkirc6tqgpnwd3blln3yfv5ckl47eg7llfxkmtovrv7c7iwohhb6ad.onion
+socks5 = true
+socks5_host = localhost
+socks5_port = 9050
```

Comment:
Previously, we were already passing [umbrel's tor proxy](https://github.com/getumbrel/umbrel/pull/1216/files#diff-7f4b716e3c9b8a48b5462a595211053098faff3253100e461ec15cceabde1c99R21-R22) here. Why was it not routing through that—because `socks5` is `false` by default?

Reply:
I think that was the problem, yes.

Reply:
Ok 👍 . I wonder how this works for BTCPay's JoinMarket setup as they [don't set this](https://github.com/btcpayserver/btcpayserver-docker/blob/master/docker-compose-generator/docker-fragments/opt-add-joinmarket.yml#L11-L22), I believe. @dennisreimann would you mind having a look over this PR as well just to be sure? A couple of extra eyes surely won't hurt here. :)